RetroBar/Languages/русский.xaml

```diff
@@ -4,6 +4,8 @@
 
     <s:String x:Key="retrobar_properties">Свойства RetroBar</s:String>
     <s:String x:Key="taskbar_options">Параметры панели задач</s:String>
+    <s:String x:Key="taskbar_appearance">Внешний вид панели задач</s:String>
+    <s:String x:Key="notification_area">Зона уведомлений</s:String>
```

Comment:
These lines do not seem to match the official translation:
![](http://yuschikev.narod.ru/Teoria/WindowsXP/ImWin.files/image007.png)